### DISCUSSION TITLE
Attention / transformer approach is on the horizon?

### DISCUSSION THREAD

**Discussion by bitdom8** [matiasvlevi/Dann#52] [General]
Thanks for the amazing lib. Do you have Attention / transformer feature?

**Comment by matiasvlevi** [comment]
> Thanks for the amazing lib. Do you have Attention / transformer feature?

Hey!

At the moment, no attention or transformer feature is implemented for Dann models.

The documentation available on https://dannjs.org represents every public component of the library, if the feature is not visible, it does not exist yet.

This would be a great possible feature to post as an Issue along with some context and relevant details.

This library is open for pull requests, so an issue can go a long way!

Thanks !

Matias Vazquez-Levi
